test/config_test.exs

```diff
@@ -51,4 +51,27 @@ defmodule CoherenceTest.Config do
     Application.put_env(:coherence, :opts, [a: [:create]])
     refute Config.has_action?(:a, :new)
   end
+
+  describe "default_routes/0" do
+    test "when are set configured globally" do
```

Comment:
Minor point... This reads a little awkward. I think the following be read a lot better:

```
describe "when default_routes/0" do
  test "are configured globally" ...
  test "are not configured" ...
end
```